src/components/elements/boxes.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+
+    <component class="ods-container__link"
+        :class="sizeBox"
+        :is="componentId"
+        v-bind="attributId">
```

Comment:
I think you could rename this variable since it is not an attribute ID but a link href ?

Reply:
@bost-h Updated !